CRM/Contact/BAO/Group.php

```diff
@@ -350,6 +350,20 @@ public static function create(&$params) {
       CRM_Utils_Hook::pre('create', 'Group', NULL, $params);
     }
 
+    // dev/core#287 Disable child groups if all parents are disabled.
+    if(!empty($params['id'])) {
+      $allChildGroupIds = self::getChildGroupIds($params['id']);
+      foreach ($allChildGroupIds as $childKey => $childValue) {
+        $parentIds = CRM_Contact_BAO_GroupNesting::getParentGroupIds($childValue);
+        $activeParentsCount = civicrm_api3('Group', 'getcount', [
+          'id' => ['IN' => $parentIds],
+          'is_active' => 1,
+        ]);
+        if (count($parentIds) == 1 || count($parentIds) > 1 && $activeParentsCount <= 1) {
```

Comment:
can this be written as `if (count($parentIds) >= 1 && $activeParentsCount <= 1) {` ?